packages/email-service/src/app/app.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { createTransport } from 'nodemailer';
+
+@Injectable()
+export class AppService {
+  transporter: ReturnType<typeof createTransport>;
+
+  constructor() {
+    console.log(process.env.NODE_ENV);
+    this.transporter = createTransport({
+      host: 'localhost',
+    });
+  }
+
+  getData(): { message: string } {
+    return { message: 'Welcome to email-service!' };
+  }
+
+  async sendOverdueNotify(receivers: string[]): Promise<boolean> {
+    try {
+      await this.transporter.sendMail({
+        from: '"Mimir App" <app@mimirapp.xyz>',
+        to: receivers.join(', '),
+        subject: 'Overdue books | Mimir App',
+        text: 'Test message',
```

Comment:
we can already make it more generic, by creating an `Email` type, which will encapsulate all email-related data (body, from, to, subj, etc). And email service should know as little as possible about books, overdues and donations.